src/System Application/App/Edit in Excel/src/Filters/EditInExcelFiltersImpl.Codeunit.al

```diff
@@ -185,6 +202,24 @@ codeunit 1491 "Edit in Excel Filters Impl."
         exit(true);
     end;
 
+    local procedure IsKey(PageNumber: Integer; ExternalTableFieldName: Text): Boolean
+    var
+        PageMetadata: Record "Page Metadata";
+        FieldMetadata: Record "Field";
+        KeyRec: Record "Key";
+    begin
+        if PageMetadata.Get(PageNumber) then begin
```

Comment:
Un-necessary identation, I would be here more exclusive. 
if not PageMetadata.Get than exit(false), than you don't surround by begin and end and code looks nicer